documentation/images/example.txt

```diff
@@ -0,0 +1 @@
+delete me
```

Comment:
Can we just remove this file?

Reply:
Yes, we can remove it. This folder was just meant to hold images for the ReadMe file. Since we do not yet have images, this folder can be removed, like the file implies.

Reply:
we can add when its needed, no need now

Reply:
file and filder strucutre is removed